src/unittests/floating_point.spec.ts

```diff
@@ -4400,17 +4405,21 @@ g.test('divisionInterval')
       })
   )
   .fn(t => {
-    const trait = FP[t.params.trait];
+    // This is a ULP based interval, so abstract should behave like f32, so
```

Comment:
Side task:   After a quick review it looked like the FP primer didn't discuss this issue (i.e. using f32 error tolerances for AbstractFloat tolerances for operations whose errors are expressed in ULPs, and why).  Please update it.

Reply:
Issue filed, https://github.com/gpuweb/cts/issues/3079